cookbooks/2D_subduction_with_two_phase_flow/fixed_slab.wb

```diff
@@ -0,0 +1,82 @@
+// Composition 1 is bound fluid
```

Comment:
I think it would be good to add comments throughout the WB file, or go into detail on each section through code snippets in the cookbook text (maybe both)?